WordPress/WordPressTest/MySiteViewModelTests.swift

```diff
@@ -1,112 +0,0 @@
-import Nimble
-@testable import WordPress
-import XCTest
-
-class MySiteViewModelTests: CoreDataTestCase {
-
-    /// When the given blog is not accessible via WordPress.com,
-    /// the section is always .siteMenu, regardless of the other inputs
-    func testGetSection_notAccessibleViaWPCom_alwaysSiteMenu() {
-        let viewModel = MySiteViewModel(coreDataStack: contextManager)
-        let blogNotAccessibleThroughWPCom = BlogBuilder(contextManager.mainContext).build()
-        // Make sure the default blog is not accessible via WordPress.com
-        expect(blogNotAccessibleThroughWPCom.isAccessibleThroughWPCom()) == false
-
-        // If the blog is not accessible via WordPress.com, then section is always .siteMenu
-        TruthTable.threeValues.forEach {
-            let section = viewModel.getSection(
-                for: blogNotAccessibleThroughWPCom,
-                jetpackFeaturesEnabled: $0,
-                splitViewControllerIsHorizontallyCompact: $1,
-                isSplitViewEnabled: $2
-            )
-            expect(section).to(equal(.siteMenu), description: "Expected .siteMenu, got \(section). Jetpack features \($0), split VC \($1), split view \($2)")
```

Comment:
These were 120 lines of tests for a function with 3 lines, so I decided it was not worth keeping it.